src/riscv/lib/src/interpreter/float.rs

```diff
@@ -580,6 +596,44 @@ impl From<RoundingMode> for Round {
     }
 }
 
+/// A trait to associate a `RoundingMode` with a specific struct.
+/// This is used for associating the correct functionality for
+/// [`run_f64_from_x64_unsigned`] and similar functions.
+pub trait RoundingModeExt {
```

Comment:
This trait deserves a more precise name. It is not an extension to rounding modes.

```suggestion
pub trait StaticRoundingMode {
```

Reply:
done [diff](https://github.com/tezos/riscv-pvm/compare/ba8654ce86ad52892058a378a909cddba3ef10f9..5a809c3f89fcbfdac87617db55cb45783b199d61)